data_services/deploy/ansible/roles/forecastio/files/data_queues/forecastio/forecastio_air.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
```

Comment:
@jbeezley Does ansible need this folder here instead of under data_services/deploy/ansible/roles/genodegp/geonodegp/data_queues, alongisde the gfms folder?  At some point next week (or sooner depending on time constraints) I'm planning to move the entire data_queues folder to it's own repo packaged as a django app (similar to https://github.com/OpenGeoscience/django-datarequests).  Do you think  this will cause problems with the ansible deployment script?